### DISCUSSION TITLE
第6节课作业(16班)

### DISCUSSION THREAD

**Discussion by Yang-Changhui** [InternLM/Tutorial#450] [作业]
16班第六节课作业提交在下面

**Comment by morcake** [comment]
https://blog.51cto.com/morcake/9355566

**Comment by etheral12138** [comment]
https://y1yc0dwhb0o.feishu.cn/docx/IGXTdK1cWo9BwrxO59LcocJ4nJd?from=from_copylink

**Comment by Dhaizei** [comment]
http://t.csdnimg.cn/whTMH

**Comment by 2404589803** [comment]
https://2404589803.github.io/My-datawhale-learning/#/shusheng_puyu_learnning53/ch06

**Comment by zszbyzsz** [comment]
https://juejin.cn/post/7327353593890095155

**Comment by hhhanqian** [comment]
http://xhslink.com/7RDG6z

**Comment by Durian-1111** [comment]
**Day06-Homework**
链接：https://xby48gs45u.feishu.cn/docx/X59BduFvmoK4CExpwzmcgjaxn3g?from=from_copylink
**6次作业笔记集合**
链接：https://xby48gs45u.feishu.cn/docx/THyMd5zMtozYRnxpouGcBZtEnsg?from=from_copylink

**Comment by Bob-Y** [comment]
### 基础作业

- 使用 OpenCompass 评测 InternLM2-Chat-7B 模型在 C-Eval 数据集上的性能

<img width="960" alt="f3c5e5eb7e89f432b4f5514ed2d4e32" src="https://github.com/InternLM/tutorial/assets/31715275/c959d5a9-c897-459d-ab9f-be37a643b3f9">

**Comment by Aitejiu** [comment]
https://www.cnblogs.com/aitejiu/p/17985010

**Comment by EnableAsync** [comment]
https://enableasync.github.io/internlm/internlm-06/
